js-core-2/lesson7.md

```diff
@@ -32,59 +32,60 @@ When we are you solving a problem, one of your main decisions as a developer is
 
 ## Object Exercises
 
-Checking if Property or Value Exists
+Exercise
+Write a function "printProperties" that takes an object and prints its properties
 
 ```js
-// Exercise
-// Write a function "printProperties" that takes an object and prints its properties
-var student = { 
-    name: "Simon", 
-    age: "21", 
+var student = {
+    name: "Simon",
+    age: "21",
     interests : ["javascript", "react"]
 };
 
 printProperties(student);
 // Output in this case should be name, age, interests
 ```
 
+Exercise
+Write a function called hasProperty that takes an object and a property
+the function should return true if the property exists, false if it doesn't
```

Comment:
The formatting for this is a bit broken: exercise should stand out a bit more. And a new sentence should start after "and a property. The function should ..."